tests/baselines/reference/classStaticBlock12.js

```diff
@@ -14,14 +14,11 @@ var __classPrivateFieldGet = (this && this.__classPrivateFieldGet) || function (
     if (typeof state === "function" ? receiver !== state || !f : !state.has(receiver)) throw new TypeError("Cannot read private member from an object whose class did not declare it");
     return kind === "m" ? f : kind === "a" ? f.call(receiver) : f ? f.value : state.get(receiver);
 };
-var C = /** @class */ (function () {
-    function C() {
-    }
-    var _a, _C_x, _C__;
-    _a = C;
-    _C_x = { value: 1 };
-    _C__ = { value: (function () {
-            __classPrivateFieldGet(C, _a, "f", _C_x);
-        })() };
-    return C;
-}());
+var _a, _C_x, _C__;
+class C {
+}
+_a = C;
+_C_x = { value: 1 };
+_C__ = { value: (() => {

```

Comment:
You can see here that `_C__` is never used, so there's no point in saving it in a local or generating the `{ value }` wrapper. The IIFE is sufficient.